contracts/Comptroller.sol

```diff
@@ -1244,6 +1245,13 @@ contract Comptroller is ComptrollerV6Storage, ComptrollerInterface, ComptrollerE
         // Update borrowers's index to the current index since we are distributing accrued COMP
         compBorrowerIndex[cToken][borrower] = borrowIndex;
 
+        if (borrowerIndex == 0 && borrowIndex > 0) {
```

Comment:
In the commit I wrote adding this increases the gas cost of claim by 12K... it's actually 27K.